### PR TITLE
issue #1540 issue_1540 empty error message was returned when student invokes MyLA before fully setup

### PR DESCRIPTION
**Pull request by zqian**

### CODE REVIEW

File: assets/src/containers/Course.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ function Course (props) {
     setSideDrawerState(open)
   }
 
-  if (error.message === 'Not Found') return (<WarningBanner>Course {courseId} has not been set up in MyLA. Contact your instructor, who can enable the visualizations by clicking on MyLA in the course navigation.</WarningBanner>)
+  if (error.message === 'Not Found' || (error.message === '')) return (<WarningBanner>Course {courseId} has not been set up in MyLA. Contact your instructor, who can enable the visualizations by clicking on MyLA in the course navigation.</WarningBanner>)
```
Comment (jonespm) [inline assets/src/containers/Course.js:58]: I think Codacy just wants this to be cleaned up to be a little more formatted like this. I can test running this though as you have it. 

```
if (error.message === 'Not Found' || (error.message === '')) {
  return (
    <WarningBanner>
      Course {courseId} has not been set up in MyLA. Contact your instructor, who can enable the visualizations by clicking on MyLA in the course navigation.
    </WarningBanner>
  )
}
```

Comment (zqian) [reply assets/src/containers/Course.js]: Got more warnings after code reformat...

**Comment by jonespm** [comment]
This seems to work but I wonder if the error is further up in the handleError method having an exception! 

image

Checking for an empty string _seems_ like it could catch the wrong error. I have a fix for this on this commit which works to return 404 if "Not Found" isn't available in the statusText.

https://github.com/tl-its-umich-edu/my-learning-analytics/commit/61a2f9990f84df279ee83d86730691851fad0be1

**Comment by zqian** [comment]
@jonespm I have tested your changes, and it is working as expected. I think yours is a better approach with both statusText and statusCode handlings. Please go ahead and merge your changes instead

**Review by jonespm — APPROVED** [review]

**Pull request merged**